c/datasetService.c

```diff
@@ -204,6 +229,19 @@ void installDatasetContentsService(HttpServer *server) {
   registerHttpService(server, httpService);
 }
 
+void installDatasetCopyService(HttpServer *server) {
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, ZSS_LOG_INSTALL_MSG, "dataset copy paste");
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG2, "begin %s\n", __FUNCTION__);
+
+  HttpService *httpService = makeGeneratedService("datasetCopy", "/datasetCopy/**");
+  httpService->authType = SERVICE_AUTH_NATIVE_WITH_SESSION_TOKEN;
+  httpService->runInSubtask = TRUE;
+  httpService->doImpersonation = TRUE;
+  httpService->serviceFunction = serveDatasetCopy;
+  httpService->paramSpecList = makeStringParamSpec("force",SERVICE_ARG_OPTIONAL, NULL);
```

Comment:
comment out?
or, try this:

if (!force && targetDatasetExists) {
  respondWithError(cannot copy, target already exists)
} else {
  doCopy
}

Reply:
I actually thought we will need `force` for the etag. But we do not. Because the new dataset will be created and we don't need to compare the etag.
And we have already checked if the target dataset exists or not in  `checkIfDatasetExists` function.